src/app/components/ConnectorForm/index.tsx

```diff
@@ -42,7 +44,28 @@ function ConnectorForm({
         </div>
         <div className="mt-16 lg:mt-0 lg:w-1/2">
           <div className="lg:flex h-full justify-center items-center">
-            <img src="assets/icons/satsymbol.svg" alt="sat" className="w-64" />
+            {video ? (
+              <div
+                className="flex-1 relative h-0"
+                style={{ paddingBottom: "56.25%" }}
+              >
+                <video
+                  className="absolute t-0 l-0 w-full h-full"
+                  width="1920"
+                  height="1080"
+                  controls
+                >
+                  <source src={video} type="video/mp4" />
+                  Your browser does not support the video tag.
```

Comment:
nit: I don't think we need the fallback? The browsers should support it anyway, or it's blank? 

but it does not matter also.

Reply:
You are right, browsers that are supported by Alby should definitely support that. I'll remove it.